addons/survey/models/survey_question.py

```diff
@@ -140,7 +140,7 @@ def default_get(self, fields):
 
     # Conditional display
     is_conditional = fields.Boolean(
-        string='Conditional Display', copy=True, help="""If checked, this question will be displayed only
+        string='Conditional Display', copy=False, help="""If checked, this question will be displayed only
```

Comment:
I wonder if this should not be done in separate commit as it seems quite right to not copy this field when duplicating a question.. (in any case) as triggering_question_id is in copy=False.
Well after reading a bit deeper, maybe add the constrains and unlink overrides also in the commit. ("Stop faking you can copy a question and pretend everything is okay" :shrug: )

You would have a commit that clean a bit the workflow of conditional questions
and another that allows to create questions in chain. (with the implied side effect that you can use them as conditional question directly)